### PR TITLE
Add nipplejs@0.6.7 w/ git auto-update

### LINKED ISSUE #11056: [Request] Add nipplejs
**Library name:** nipplejs
**Git repository url:** https://github.com/yoannmoinet/nipplejs
**npm package name or url** (if there is one): nipplejs 
**License (List them all if it's multiple):** MIT
**Official homepage:** http://yoannmoinet.github.io/nipplejs
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by ajay2507**
close #11056, cc @yoannmoinet

Pull request for issue: #11056

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/yoannmoinet/nipplejs.git
 * Official website (optional, not the repository): https://yoannmoinet.github.io/nipplejs/
 * NPM package url (optional):  https://www.npmjs.com/package/nipplejs
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 21
   - Count of stars: 774
   - Count of forks: 51
 * NPM download stats (optional):
   - Downloads in the last day: 13
   - Downloads in the last week: 79
   - Downloads in the last month: 440

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/nipplejs/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "version": "0.6.7",
+  "filename": "nipplejs.min.js",
+  "description": "A virtual joystick for touch capable interfaces",
+  "author": "Yoann Moinet <yoann.moinet@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yoannmoinet/nipplejs.git"
+  },
+  "homepage": "http://yoannmoinet.github.io/nipplejs",
```
Comment (sufuf3) [inline ajax/libs/nipplejs/package.json]: Let's use `https`
Using https is better because it's more security for browsing.

**Comment by extend1994** [comment]
Hi @ajay2507 Do you want to continue to work on this?

**Comment by ajay2507** [comment]
@extend1994  i was busy in some other things, I will complete it today.

**Comment by extend1994** [comment]
@ajay2507 Don't feel stressed here, it's glad to hear that you want to keep doing this! Just take your time and ask us if you need help 😃

**Comment by sufuf3** [comment]
Hi @ajay2507 Do you want to continue to work on this?

I think this PR almost finish, except for homepage part.

Thank you.

**Comment by ajay2507** [comment]
Hi @sufuf3 sorry for the delay. Ya its almost finished. I will do it today for sure.

**Comment by ajay2507** [comment]
created new pull request https://github.com/cdnjs/cdnjs/pull/11823

**Comment by PeterDaveHello** [comment]
@ajay2507 thanks.

**Pull request closed without merging**